js/script.js

```diff
@@ -0,0 +1,149 @@
+(function(){
+    var days = ['Sun','Mon','Tue','Wed','Thur','Fri','Sat'];
+    var months = ['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec'];
+
+    var pastEventsQuery = "https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_id=17778422&only=time%2Cevent_url%2Cname%2Cdescription%2Cyes_rsvp_count%2Crsvp_limit&photo-host=secure&page=40&fields=&order=time&status=past&desc=false&sig_id=153356042&sig=7c33ad50321a70d19f843d60c0c2eaee08b67d09";
+    var upcomingEventsQuery = "https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_id=17778422&only=time%2Cevent_url%2Cname%2Cdescription%2Cyes_rsvp_count%2Crsvp_limit&photo-host=secure&page=20&fields=&order=time&desc=false&status=upcoming&sig_id=153356042&sig=84e9ac6ce37bdb3c00e4f82fe5a7ce798865fbe4";
+    var membersQuery = "https://api.meetup.com/2/members?offset=0&format=json&group_id=17778422&only=photo%2Cname%2Clink&photo-host=secure&page=200&order=name&sig_id=153356042&sig=4d8e3265b4374b84aabb8efcc26eb8107a3ec81b";
+
+    var isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
+
+    var pastEvents = [],
+        upcomingEvents = [],
+        members = [];
+
+    // Get upcoming events
+    $.ajax({
+      url: upcomingEventsQuery + "&offset=0",
+        type: "GET",
+        cache: false,
+        dataType: "jsonp",
+        crossDomain: true,
+        success: function(data){
+            // Currently will only get the first 20
+            upcomingEvents = data.results;
+            populateUpcomingEvents();
+        }
+    });
+    // Get past events
+    $.ajax({
+      url: pastEventsQuery + "&offset=0",
+        type: "GET",
+        cache: false,
+        dataType: "jsonp",
+        crossDomain: true,
+        success: function(data){
+            // Currently will only get the first 40
+            pastEvents = data.results;
+            populatePastEvents();
+        }
+    });
+    if(!isMobile) {
+        // Get members, but only for desktop (don't want to waste peoples money)
```

Comment:
This regex isn't going to catch every mobile user - there's a more complete version [here](http://detectmobilebrowsers.com/) - though even with that; it's not a very good approach to infer data plans based on it, or to tweak content as a result of that (removing images for loading time is a whole different thing).

Something that would be cooler would be to use media queries to work out wether to request images (or a sub-set, which would be super-cool as mobile users would still be able to see some faces).  You could do that in this script with [matchmedia](http://caniuse.com/#search=matchmedia), something like:

```js
// replacing `isMobile`
var isSmall = window.matchMedia && window.matchMedia('(max-width: 400px)').matches
```

This is cool because it kind of covers your idea of protecting data plans for mobile devices, but does it with a more consistent property of the browser.